<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑體 TW Regular</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>151</width>
      <height>158</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_port">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_port">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_baudrate">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_baudrate">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_data_len">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>數據位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_data_len">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_stop_len">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_stop_len">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_parity">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_parity">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_flow_ctrl">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> qproperty-wordWrap: true;</string>
         </property>
         <property name="text">
          <string>流控</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_flow_ctrl">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="push_button_open_close">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿点阵正黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_led">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>155</width>
      <height>192</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_tx_filesource">
       <property name="font">
        <font>
         <family>思源黑體 TW Regular</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>啟動文件數據源</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tx_hex_disp">
       <property name="font">
        <font>
         <family>思源黑體 TW Regular</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>按十六進制顯示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tx_new_line">
       <property name="font">
        <font>
         <family>思源黑體 TW Regular</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>發送新行即附加位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tx_rts">
       <property name="font">
        <font>
         <family>思源黑體 TW Regular</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>數據流循環接收RTS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tx_dtr">
       <property name="font">
        <font>
         <family>思源黑體 TW Regular</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>數據流循環發送DTR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tx_cycle">
       <property name="font">
        <font>
         <family>思源黑體 TW Regular</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>定時發送</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_tx_cycle_timer"/>
       </item>
       <item>
        <widget class="QLabel" name="label_tx_cycle_unit">
         <property name="text">
          <string>ms/次</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_tx_load_file">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>文件載入</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>發送區設置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>串口設置</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_tx">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>371</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tx_clear">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tx_enable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>發送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tx_count">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>540</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>發送：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recv_count">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>540</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>接收：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tx_clear_count">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>復位計數</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>發送區</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_tx">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tx_hex">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑體 TW Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>十六進制</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_rx">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>471</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_rx">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清空接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_rx">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存接收</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>接收區</string>
    </property>
    <widget class="QCheckBox" name="checkBox_rx_pause">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑體 TW Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>暫停顯示</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_rx_hex_disp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑體 TW Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>接收十六进制显示</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_rx_newline_disp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑體 TW Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>自动换行显示</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_rx_resource">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑體 TW Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>接收转向文件</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>接收区设置</string>
    </property>
   </widget>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>push_button_open_close</zorder>
   <zorder>label_led</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>textBrowser_tx</zorder>
   <zorder>pushButton_tx_clear</zorder>
   <zorder>pushButton_tx_enable</zorder>
   <zorder>label_tx_count</zorder>
   <zorder>label_recv_count</zorder>
   <zorder>pushButton_tx_clear_count</zorder>
   <zorder>textBrowser_rx</zorder>
   <zorder>pushButton_clear_rx</zorder>
   <zorder>pushButton_save_rx</zorder>
   <zorder>checkBox_rx_hex_disp</zorder>
   <zorder>checkBox_rx_newline_disp</zorder>
   <zorder>checkBox_rx_resource</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_option">
    <property name="title">
     <string>選項</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_helo">
    <property name="title">
     <string>幫助 </string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_language">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="action_english"/>
    <addaction name="action_chinese_s"/>
    <addaction name="action_chinese_t"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_option"/>
   <addaction name="menu_helo"/>
   <addaction name="menu_language"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_english">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_chinese_s">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="action_chinese_t">
   <property name="text">
    <string>繁體中文</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
